src/js/utils.js

```diff
@@ -22,6 +22,10 @@ export async function getErc20Data (address) {
     }),
     bridgedNep141.getMetadata(address, window.nearUserAddress)
   ])
+  if (address.toLowerCase() === '0x9f8f72aa9304c8b593d555f12ef6589cc3a579a2') {
+    erc20.name = 'MKR'
+    nep141.name = 'nMKR'
+  }
```

Comment:
Why should be handled in such an special way?

Reply:
https://etherscan.io/address/0x9f8f72aa9304c8b593d555f12ef6589cc3a579a2#readContract
The Maker token doesn't use standard erc20 symbol and name, it returns 32 bytes instead of the string ticker/name.

Reply:
I see, it returns the name hex encoded :(